### PR TITLE
Add cluster marker

### PR DESCRIPTION
**Pull request by Skyriixxx**

### CODE REVIEW

File: src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      position,
+      icon: {
+        url: `data:image/svg+xml;base64,${svg}`,
+        scaledSize: new google.maps.Size(75, 75),
+      },
+      label: {
+        // display markers number in the cluster
+        text: String(count),
+        color: '#05427d',
+        fontSize: '12px',
```
Comment (LucasBrazi06) [inline src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts:23]: Increase font to 18px

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

File: src/app/pages/charging-stations/charging-stations-map/charging-stations-map.component.ts [hunk 2]
```diff
@@ -59,20 +64,23 @@ export class ChargingStationsMapComponent implements OnInit {
...
+                url: this.getIconColorAccordingToConnectorStatus(chargingStation.connectors),
+                labelOrigin: { x: 11, y: -10 } as google.maps.Point
+              }
+            })
+          );
           // Extends the Map bounds
-          bounds.extend(latLng);
+          this.bounds.extend(latLng);
         }
+        new MarkerClusterer({map: this.googleMapService.googleMap, markers: this.markers, renderer: new CustomClusterRenderer()});
```
[the remark this answers is not in the document]
Comment (Skyriixxx) [reply src/app/pages/charging-stations/charging-stations-map/charging-stations-map.component.ts]: done

File: src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { Cluster, ClusterStats, Renderer } from '@googlemaps/markerclusterer';
+
+export class CustomClusterRenderer implements Renderer {
```
[the remark this answers is not in the document]
Comment (Skyriixxx) [reply src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts]: done

File: src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts (hunk 1 above)
Comment (Skyriixxx) [reply src/app/pages/charging-stations/charging-stations-map/custom-cluster-renderer.ts]: done

**Review by LucasBrazi06 — APPROVED** [review]
GUT

**Pull request merged**